### PR TITLE
Type carousel

### PR DESCRIPTION
**Pull request by Dash293**

### CODE REVIEW

**Comment by NtGZeko** [comment]
is good for me

**Review by NtGZeko — DISMISSED** [review]
I haz a problem

File: frontend/src/style.jsx [hunk 1]
```diff
@@ -42,6 +42,8 @@ export default styled.div`
   color: var(--color-font-general);
   box-sizing: border-box;
 
+
```
Comment (LuigiMenez) [inline frontend/src/style.jsx:45]: Warning a blank line

File: frontend/src/components/Header/style.jsx [hunk 2]
```diff
@@ -12,7 +12,9 @@ export default styled.header`
     text-decoration: none;
   }
   .navBar {
+
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: a blank line

File: frontend/src/components/Header/style.jsx [hunk 3]
```diff
@@ -12,7 +12,9 @@ export default styled.header`
...
     height: 10vh;
+
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: blank line

File: frontend/src/components/Header/style.jsx [hunk 4]
```diff
@@ -27,8 +29,10 @@ export default styled.header`
     transition: 0.5s;
   }
   .navBarSolid {
+
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: blank line

File: frontend/src/components/Header/style.jsx [hunk 5]
```diff
@@ -27,8 +29,10 @@ export default styled.header`
...
     background-color: rgb(8, 7, 8, 0.8);
     color: var(--font-primaire);
+
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: blank line

File: frontend/src/components/Header/style.jsx [hunk 6]
```diff
@@ -71,6 +75,7 @@ export default styled.header`
   .navBarLink {
     display: block;
     padding: 1.5rem;
+
```
Comment (LuigiMenez) [inline frontend/src/components/Header/style.jsx]: blank line

File: frontend/src/components/TypeCarroussel/style.jsx [hunk 7]
```diff
@@ -1,21 +1,42 @@
 import styled from "styled-components";
 
 export default styled.section`
-  height: 40vh;
-  max-width: 800px;
+  width: 80vw;
+
```
Comment (LuigiMenez) [inline frontend/src/components/TypeCarroussel/style.jsx]: blank line

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Remove the blank line in css

**Review by NtGZeko — DISMISSED** [review]

**Review by LuigiMenez — DISMISSED** [review]
Ok for me

File: frontend/src/App.jsx [hunk 8]
```diff
@@ -9,12 +10,15 @@ import SGlobal from "./style";
 function App() {
   return (
     <SGlobal>
+
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/suggestion" element={<Suggestion />} />
+         <Route path="/results/:category" element={<ResultsCategory />} />
```
Comment (LoicBrassart) [inline frontend/src/App.jsx:16]: This line looks like Prettier is misconfigured :thinking:

**Review by LoicBrassart — DISMISSED** [review]
I don't know what's the problem with Prettier's conf, but most of the project is well-indented so I'll let it slide

**Review by NtGZeko — APPROVED** [review]

**Review by Elfaram — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**